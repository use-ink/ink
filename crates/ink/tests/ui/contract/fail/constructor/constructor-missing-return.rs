#![allow(unexpected_cfgs)]

#[ink::contract]
mod contract {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(constructor)]
        pub fn constructor() {}

        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
