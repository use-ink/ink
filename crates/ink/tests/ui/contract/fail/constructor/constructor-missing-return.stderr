error: missing return for ink! constructor
  --> tests/ui/contract/fail/constructor-missing-return.rs:10:13
   |
10 |         pub fn constructor() {}
   |             ^^^^^^^^^^^^^^^^
